docs/README.md

```diff
@@ -9,12 +9,14 @@ FlowKit's documentation is built using [Mkdocs](http://mkdocs.org).
 - Docker
 - Pipenv
 
-To build the docs, first install the necessary dependencies by running `pipenv install` (in the `docs/` subfolder). Once this completes, you can build the docs by running `pipenv run build`.
+To build the docs, first install the necessary dependencies by running `pipenv install` (in the `docs/` subfolder). Once this completes, you can build the docs by running `pipenv run build`, after sourcing the top level `development_environment` file.
```

Comment:
Sourcing `development_environment` is not sufficient - this will only set shell variables, not export them as env vars.

Reply:
set -a &&. ./development_environment && set +a is what’s actually needed I think 🤔

Reply:
Yep, that works. Probably worth mentioning this in the docs.